templates/single-profile.twig

```diff
@@ -92,29 +92,31 @@
             {{post.content}}
         </div>
     </div>
-    <div class="paragraph-widget paragraph-widget--article-category-tags">
-        <h4 class="category-title">{{ tax_a[0].name }}</h4>
-        <ul class="category-tags">
-            {% for term in post.terms %}
-                    {% if term.taxonomy == tax_a[0].taxonomy %}
-                        <li>
-                            <a href="{{ term.link|e('esc_url') }}">{{ term.name }}</a>
-                        </li>
-                    {% endif %}
-            {% endfor %}
-        </ul>
-        <h4 class="category-title">{{ tax_b[0].name }}</h4>
-        <ul class="category-tags">
-            {% for term in post.terms %}
-                {% if term.taxonomy == tax_b[0].taxonomy %}
-                    <li>
-                        <a href="{{ term.link|e('esc_url') }}">{{ term.name }}</a>
-                    </li>
+        <div class="paragraph-widget paragraph-widget--article-category-tags">
+            {% if tax_a[0].name %}
```

Comment:
Where is this `tax_a` variable coming from? I'm not seeing it when I pull in this commit.